app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsItemFragment.kt

```diff
@@ -71,6 +72,13 @@ class SuggestedEditsCardsItemFragment : SuggestedEditsItemFragment() {
         when (parent().action) {
             TRANSLATE_DESCRIPTION -> {
                 disposables.add(EditingSuggestionsProvider.getNextArticleWithMissingDescription(WikiSite.forLanguageCode(parent().langFromCode), parent().langToCode, true)
+                        .map {
+                            if (it.second.description.isNullOrEmpty()) {
```

Comment:
@cooltey During translation. suggestion, wont this always be empty?

Reply:
You can notice from the code below:
```
                            val source = pair.second
                            val target = pair.first
```